core/shared/src/main/scala/zio/clock/package.scala

```diff
@@ -46,46 +46,58 @@ package object clock {
     }
 
     object Service {
-      val live: Service = new Service {
```

Comment:
Let's please not change the implementation of the live service but just provide this as an additional layer.

Reply:
restored